core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/BillingApiServiceV2.scala

```diff
@@ -57,25 +66,19 @@ trait BillingApiServiceV2 extends UserInfoDirectives {
           pathEndOrSingleSlash {
             get {
               parameters(
-                "startDate".as[String],
-                "endDate".as[String],
+                "startDate".as[DateTime],
```

Comment:
What happens now with invalid dates?

Reply:
It throws a 400. The error message is slightly different, but still informative

```
"400 Bad Request": "The query parameter 'endDate' was malformed:\nInvalid format: \"bad-date\""